src/jabby/server/public.d.ts

```diff
@@ -1,5 +1,7 @@
 import { World } from "../modules/types"
+import SchedulerExports from "./scheduler"
 
-declare const public: World[] & { updated: boolean }
+type Scheduler = typeof SchedulerExports.schedulers[number];
```

Comment:
This is dodgy. Feel free to move the `Scheduler` class and `ProcessingFrame` interface from `src/jabby/server/scheduler.d.ts` to `src/jabby/modules/types.d.ts` and then import them in `src/jabby/server/scheduler.d.ts`

Reply:
👍